_editorial_policies/02_editor_instructions.md

```diff
@@ -11,23 +11,26 @@ This provides instructions to editors handling manuscripts for LiveCoMS.
 
 ## Workflow overview
 
-Authors contact LiveCoMS (specifically, the Lead Editor in a particular subject area) with a presubmission letter, proposing an article in a particular area.
-Once this is approved, they proceed with article preparation and ultimately submit it to that section of LiveCoMS.
-The Lead Editor then checks that the article is consistent with what was proposed in the presubmission letter, and then assigns it to an appropriate Associate Editor who manages the review process and makes a final decision on the manuscript, reporting the final decision to the authors.
+Authors contact LiveCoMS (specifically, the Section Lead Editor in that topic) with a presubmission letter, proposing an article in a particular area. The Section Lead Editor assigns this presubmission letter to and Associate Editor, who works with the authors to revise the article plans to be suitable for LivCoMS, and the Section Lead Editor signs off on the final plans.
```

Comment:
"Section Lead editor in that topic" seems like odd phrasing; "responsible for that topic" or "in that particular subject area" seems better.